docker/inference/dockerfile.ctr

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1.2
-ARG TRITON_VERSION=22.03
+ARG TRITON_VERSION=22.0
```

Comment:
Is this the right triton version?

Reply:
its not but these files are irrelevant and need to be removed anyway